source/openstack.rst

```diff
@@ -173,6 +173,20 @@ Pre-requisites
 
 #. You need to install `mod_auth_openidc <https://github.com/pingidentity/mod_auth_openidc>`_ for adding support for OpenID Connect to Apache.
 
+.. note::
+   EGI expects that your Keystone accepts clients with certificates from the
```

Comment:
So here it's only EGI monitoring? Can we update?

Reply:
I rephrased, hope is better now

Reply:
Thanks@